src/idiomatic/foundations-api-design.md

```diff
@@ -0,0 +1,3 @@
+---
+minutes: 2
+---
```

Comment:
Do you intend to add something here?

Compare: https://github.com/google/comprehensive-rust/blob/main/src/idiomatic/leveraging-the-type-system.md?plain=1

At the very least, a title and `{{%segment outline}}` are needed.